### ISSUE TITLE
Details column (nb views) keeps incrementing when same user revisits a thread

### ISSUE THREAD

**Issue by timotheecour** [nim-lang/nimforum#121]
Details column (nb views) keeps incrementing when same user revisits a thread (or simply refreshes the page); eg; refreshing https://forum.nim-lang.org/t/3750 3 times increments by 3.
this makes the nb views rather meaningless. Instead, should count for eg number of unique logged in users that visited it (or, but it's trickier, nb unique ip addresses, but that's more complex to get it right and less robust)

**Comment by dom96** [comment]
> this makes the nb views rather meaningless

I disagree. As long as people don't abuse this of course, so far they haven't :)
